crates/pipeline-types/src/format/parquet.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
+const fn default_buffer_size_records() -> usize {
+    100_000
+}
+
+#[derive(Deserialize, Serialize, ToSchema)]
+pub struct ParquetParserConfig {}
+
+#[derive(Deserialize, Serialize, ToSchema)]
+pub struct ParquetEncoderConfig {
+    /// Number of records before a new parquet file is written.
```

Comment:
Since this will appear in swagger docs, the comment should explicitly state the default value.